e2e/tests/e2e.rs

```diff
@@ -58,10 +58,14 @@ fn e2e_channel_success(channel: &'static str, arch: &'static str) {
     if let Err(e) = cincinnati::testing::compare_graphs_verbose(
         expected,
         actual,
-        &[
-            "io.openshift.upgrades.graph.previous.remove_regex",
-            "io.openshift.upgrades.graph.previous.remove",
-        ],
+        cincinnati::testing::CompareGraphsVerboseSettings {
+            unwanted_metadata_keys: &[
+                "io.openshift.upgrades.graph.previous.remove_regex",
+                "io.openshift.upgrades.graph.previous.remove",
+            ],
+
+            ..Default::default()
+        },
```

Comment:
@steveeJ I am not sure what is this for? Can you add more context to the commit?

Reply:
This came from https://github.com/openshift/cincinnati/pull/312/commits/4cefba246adc9d37ef8965634c405e7a1b569a6d but still a bit explanation/documentation/comment will help.

Reply:
It effectively allows us to omit arguments to the `graph_compare_graph` and leaving them at their default.

Reply:
We don't want to include metadata with the given keys in the graph comparison, so we remove these before comparing the graphs.